lmfit/parameter.py

```diff
@@ -284,6 +284,53 @@ def pretty_print(self, oneline=False, colwidth=8, precision=4, fmt='g',
             print(line.format(name_len=name_len, n=colwidth, p=precision,
                               f=fmt, **pvalues))
 
+    def _repr_html_(self):
+        """Returns a HTML representation of parameters data."""
+        # We omit certain columns if they would be empty.
+        any_err = any([p.stderr is not None for p in self.values()])
+        any_par_expr = any([p.expr is not None for p in self.values()])
+        # Helper functions for shorter code
+        html = []
+        add = html.append
+        cell = lambda x: add('<td>%s</td>' % x)
+        head = lambda x: add('<th>%s</th>' % x)
+
+        add('<table>')
+        # Header ----
+        add('<tr>')
+        headers = ['name', 'value', '(init)', 'min', 'max', 'vary']
+        if any_err:
+            headers = headers[:2] + ['', 'error', 'rel. error'] + headers[2:]
+        if any_par_expr:
+            headers.append('expr.')
+        for h in headers:
+            head(h)
+        add('</tr>')
+        # Parameters ----------
+        for p in self.values():
+            add('<tr>')
+            cell(p.name)
+            cell('%f' % p.value)
+            if p.stderr is not None:
+                cell('+/-')
+                cell('%f' % p.stderr)
+                cell('%.2f%%' % (100 * p.stderr / p.value))
+            elif any_err:
+                cell('')
+                cell('')
+                cell('')
+            cell(p.user_value)
```

Comment:
this causes problems, in the example below:
```
import lmfit
p = lmfit.Parameters()
p.add_many(('a1', 4), ('a2', 4), ('t1', 3), ('t2', 3., True))

p
```

with the following traceback

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/IPython/core/formatters.py in __call__(self, obj)
    343             method = get_real_method(obj, self.print_method)
    344             if method is not None:
--> 345                 return method()
    346             return None
    347         else:

~/Library/Python/3.7/lib/python/site-packages/lmfit-0.9.12+82.gd464830-py3.7.egg/lmfit/parameter.py in _repr_html_(self)
    320                 cell('')
    321                 cell('')
--> 322             cell(p.user_value)
    323             cell('%f' % p.min)
    324             cell('%f' % p.max)

AttributeError: 'Parameter' object has no attribute 'user_value'
````